test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1395,3 +1604,42 @@ def tearDown(self):
         # Disable the cgroups hook
         conf = {'enabled': 'False', 'freq': 30, 'event': events}
         self.server.manager(MGR_CMD_SET, HOOK, conf, self.hook_name)
+        # Cleanup the frozen jobs
+        fdir = self.paths['freezer']
+        if os.path.isdir(fdir):
+            fpath = os.path.join(fdir, 'PtlPbs')
+            if os.path.isdir(fpath):
+                jid = os.listdir(fpath)
+                if jid:
+                    for file in jid:
+                        self.logger.info("found jobdir %s" % file)
+                        jpath = os.path.join(fpath, file)
+                        freezer_file = os.path.join(jpath, 'freezer.state')
+                        # Thaw the cgroup
+                        state = 'THAWED'
+                        fn = self.du.create_temp_file(
+                            hostname=self.momA, body=state)
+                        ret = self.du.run_copy(hosts=self.momA, src=fn,
+                                               dest=freezer_file, sudo=True,
+                                               uid='root', gid='root',
+                                               mode=0644)
+                        os.remove(fn)
+                        cmd = "rmdir " + jpath
+                        self.logger.info("deleting jobdir %s" % cmd)
+                        # This is currently blocked due to infrastructure
```

Comment:
Add the PP number to this comment.

Reply:
This is not a generic issue but an issue on my test system. once I get to test on a non-restricted system I will remove this comment.